google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClientTests.java

```diff
@@ -204,6 +212,54 @@ public void testCreateTable() {
     assertThat(result).isEqualTo(Table.fromProto(expectedResponse));
   }
 
+  @Test
+  public void testCreateTableWithDeletionProtection() {
```

Comment:
This test doesn't seem to create a table with deletion protection, or maybe I'm missing something?

Reply:
yes great call, thanks :D